R/importRDB1.r

```diff
@@ -9,7 +9,7 @@
 #' @param asDateTime logical, if \code{TRUE} returns date and time as POSIXct, if \code{FALSE}, Date
 #' @param tz character to set timezone attribute of datetime. Default is an empty quote, which converts the 
 #' datetimes to UTC (properly accounting for daylight savings times based on the data's provided tz_cd column).
-#' Possible values to provide are "America/New_York","America/Chicago", "America/Denver","America/Los_Angeles",
+#' Possible values to provide include "America/New_York","America/Chicago", "America/Denver","America/Los_Angeles",
```

Comment:
Maybe also point people to OlsonNames and/or https://en.wikipedia.org/wiki/List_of_tz_database_time_zones for the complete list? I always have to search around for those. 

To maintain emphasis on the America options, you could even change the above to say that you *recommend* those.

And maybe add UTC to affirm it as one of the recommended options?

```
Recommended values to provide are "UTC","America/New_York","America/Chicago", 
"America/Denver","America/Los_Angeles","America/Anchorage","America/Honolulu",
"America/Jamaica","America/Managua","America/Phoenix", or "America/Metlakatla".
See https://en.wikipedia.org/wiki/List_of_tz_database_time_zones for a complete list of possibilities.
```